assets/constants.ts

```diff
@@ -7057,6 +7057,6 @@ export default {
   ],
   "CONTRIBUTOR_CERTIFICATE_WIDTH": 1493,
   "CONTRIBUTOR_CERTIFICATE_HEIGHT": 1313,
-  "BRANCH_NAME": "",
-  "SHORT_COMMIT_HASH": ""
+  "BRANCH_NAME": "log-data",
```

Comment:
Can you explain this change?

Reply:
Sir this change is automatically made when I created a new branch (name : log-data) and then commit and push code to that new branch.

Reply:
@JayVivarekar now can you review this? This is automatically generated when we make a new branch for pr.

Reply:
Please undo changes in this file.